nextcord/ext/commands/cog.py

```diff
@@ -161,27 +112,9 @@ def __new__(cls: Type[CogMeta], *args: Any, **kwargs: Any) -> CogMeta:
                     if elem.startswith(("cog_", "bot_")):
                         raise TypeError(no_bot_cog.format(base, elem))
                     commands[elem] = value
-                elif asyncio.iscoroutinefunction(value):
-                    try:
-                        getattr(value, "__cog_listener__")
-                    except AttributeError:
-                        continue
-                    else:
-                        if elem.startswith(("cog_", "bot_")):
-                            raise TypeError(no_bot_cog.format(base, elem))
-                        listeners[elem] = value
-
-        new_cls.__cog_commands__ = list(commands.values())  # this will be copied in Cog.__new__
-
-        listeners_as_list = []
-        for listener in listeners.values():
-            for listener_name in listener.__cog_listener_names__:
-                # I use __name__ instead of just storing the value so I can inject
-                # the self attribute when the time comes to add them to the bot
-                listeners_as_list.append((listener_name, listener.__name__))
-
-        new_cls.__cog_listeners__ = listeners_as_list
-        return new_cls
+
+        new_cls.__cog_commands__ = list(commands.values())  # type: ignore
+        return new_cls  # type: ignore
```

Comment:
consider adding comments here explaining why `# type: ignore` is necessary and the type issues cannot be resolved